meson.build

```diff
@@ -7,7 +7,3 @@ wraps = get_option('wraps')
 foreach w : wraps
   subproject(w)
 endforeach
-
-run_target('update',
-  command: ['meson', 'subprojects', 'update', '--sourcedir', meson.current_source_dir(), '--reset'] + wraps,
-)
```

Comment:
Why are we deleting this?

Reply:
Otherwise it fails because in the CI job "meson" is not in PATH, and I thought nobody would be using that anyway. IIRC we have a PR somewhere to get meson cmd_array that could fix this.